doc/DESI_SPECTRO_DATA/NIGHT/EXPID/desi-EXPID.rst

```diff
@@ -16,138 +16,302 @@ There is one HDU per spectrograph camera with EXTNAMEs like
+EXPID    1000                                                       int   Exposure number
+EXPFRAME 0                                                          int   Frame number
+FILENAME /exposures/desi/sps/20190220/00001000/sp0-00001000.fits.fz str   Name of
+FLAVOR   zero                                                       str   Observation type
+REQTIME  0.0                                                        float [s] Requested exposure time
+EXPTIME  0.0                                                        float [s] Actual exposure time
+OBSID    kp4m20190220t212702                                        str   Unique observation identifier
+TIMESYS  UTC                                                        str   Time system used for date-obs
+NIGHT    20190220                                                   str   Observing night
+DATE-OBS 2019-02-20T21:27:02.730679                                 str   [UTC] Observation data and start time
+TIME-OBS 21:27:02.730679                                            str   [UTC] Observation start time
+MJD-OBS  58534.894                                                  float Modified Julian Date of observation
+ST       00:02:43.730                                               str   Local Sidereal time at observation start (HH:MM
+SPECGRPH SP0                                                        str   Spectrograph name
+MODULE   SPECTROGRAPHS                                              str   Image Sources/Component
+DOSVER   1.001                                                      float DOS software version
+DEVICES  SPECTCON0,CCDS0Z,CCDS0R,CCDS0B                             str
+CHECKSUM 2aAn3a3k2a9k2a9k                                           str   HDU checksum updated 2019-02-20T21:27:57
+DATASUM           0                                                 str   data unit checksum updated 2019-02-20T21:27:57
+TELRA    335.03                                                     float Telescope pointing RA [degrees] **MISSING**
+TELDEC   19.88                                                      float Telescope pointing dec [degrees] **MISSING**
+AIRMASS  1.0                                                        float Airmass at middle of exposure **MISSING**
+TILEID   4                                                          int   DESI tile ID **MISSING**
+FEEVER   SIM                                                        str   **MISSING**
+DETECTOR SIM                                                        str   **MISSING**
+DEPNAM00 python                                                     str   **MISSING**
+DEPVER00 3.5.2                                                      str   **MISSING**
+======== ========================================================== ===== ===============================================
 
 Empty HDU.
 
-TODO:
+HDU1
+----
 
-  * Synchronize with ICS for keywords (e.g. FLAVOR -> PROGRAM)
+EXTNAME = SPECTCON0
```

Comment:
See above comment about SPECTCONn vs. SPECTCONS

Also note that this is HDU 1 for the current FV data, but may be the last HDU of the fully merged desi*.fits.fz files.  i.e. data model validation shouldn't get tripped up if this appears in a different order.

Reply:
Noted.